core/PRP/Modifier/plPythonFileMod.cpp

```diff
@@ -222,6 +222,15 @@ unsigned int plPythonParameter::PlasmaToMapped(unsigned int type, PlasmaVer ver)
         case 26: return kNone;
         default: return 0;
         }
+    } else if(ver.isMoul()) {
+        switch (type) {
+        case 20: return kClusterComponent;
+        case 21: return kMaterialAnimation;
+        case 22: return kGrassShaderComponent;
+        case 23: return kLayer;
+        case 24: return kNone;
```

Comment:
Will this break anything with older MOUL PRPs?

Reply:
Not that I'm aware of, but I was hoping those with better knowledge of it would know for sure.  I don't think a None-type attribute exists in any PRPs, but I could be wrong.

It's really difficult to be certain about changes in versioned data files when MOULa shares versions with other MOULas.